webapp-python/build-tools/python-build-local.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+#This helper script will package the Python project in
+# webapp-python, build it locally, package it as a zip
+# file for Lambda, and upload that zip file to an S3 bucket.
+#
+#Prerequisites:
+# 1. Python 3.6 available locally.
+# 2. Local environment MUST be a recent version of Amazon Linux.
```

Comment:
Really? Why? What's a "recent version"?

Reply:
Because we (at least potentially) need to compile stuff with static linking to system binaries. To be fair, I haven't tested this since before pyca/cryptography 2.0.0, but to be safe the build environment should be as close as possible to the Lambda runtime.

As far as "how recent"...currently, best would be something based on the `amzn-ami-hvm-2017.03.1.20170812-x86_64-gp2` AMI.

https://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html